LiteCore/Query/N1QL_Parser/n1ql.leg

```diff
@@ -315,7 +315,10 @@ propertyName
 
 
 function =
-    f:functionName e:parenExprs         { $$ = insertAny(e, 0, f.as<string>() + "()");}
+    "meta"i _ '(' _                    { f = op("meta");}
```

Comment:
Please use `META` instead. We try to declare the reserved words only once.

Reply:
I tried to make a point that, like all other function names, "meta" is not a reserved word. In fact, function names, or, IDENTITY for that matter, cannot be reserved words. I feel that "meta" here is merely a function name. In fact, as the next step to remove meta property path, I planned to take META from reserved words.